toolchain/lower/handle.cpp

```diff
@@ -333,7 +333,7 @@ auto HandleInst(FunctionContext& context, SemIR::InstId inst_id,
 
 auto HandleInst(FunctionContext& context, SemIR::InstId inst_id,
                 SemIR::VtablePtr inst) -> void {
-  context.SetLocal(inst_id, context.GetValue(inst.vtable_id));
+  context.SetLocal(inst_id, context.GetVtable(inst.vtable_id));
```

Comment:
Is this reachable? I'd expect that we never see a non-constant `vtable_ptr` instruction in lowering, and could `CARBON_FATAL` here instead.